Alps code dev

ly120983 opened the pull request:
change alps code gen to ir

ly120983 committed:
test_commit

Commit:
[alps code gen]

Commit:
[删除无用代码]

typhoonzero commented:
To apply the new alps codegen, you need to update `executor_ir.go` to insert below code above https://github.com/sql-machine-learning/sqlflow/blob/develop/pkg/sql/executor_ir.go#L157:

```go
if os.Getenv("SQLFLOW_submitter") == "alps" {
    code, err := alps.Train(trainIR)
}
```

Then the generated alps python code will be executed.

Commit:
[merge]
Merge branch 'develop' of https://github.com/ly120983/sqlflow into alps_code_dev

Commit:
修改ci相关的问题

Commit:
修改ci相关的问题

tonyyang-svail commented:
Close to due lack of activity.
